packages/ember-routing-htmlbars/tests/helpers/render_test.js

```diff
@@ -205,14 +148,14 @@ test("{{render}} helper should render given template with a supplied model", fun
 
   Ember.TEMPLATES['post'] = compile("<p>{{model.title}}</p>");
 
-  appendView(view);
+  runAppend(view);
 
   var postController = view.get('_childViews')[0].get('controller');
 
   equal(view.$().text(), 'HIRails is omakase');
   equal(postController.get('model'), post);
 
-  set(controller, 'post', { title: "Rails is unagi" });
+  runSet(controller, 'post', { title: "Rails is unagi" });
```

Comment:
Strike that, I see that you essentially just renamed this (to be MUCH more appropriate).